ci-operator/step-registry/codeready-toolchain/devsandbox-dashboard-prod/codeready-toolchain-devsandbox-dashboard-prod-commands.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+export KUBECONF=/tmp/kubeconfig
+printf '%s' "${KUBECONFIG}" | base64 -d > "${KUBECONF}"
```

Shellcheck (0.11.0)

[info] Possible misspelling: KUBECONFIG may not be assigned. Did you mean KUBECONF?

(SC2153)